server/TypeDBServer.java

```diff
@@ -61,27 +62,28 @@
 import static com.vaticle.typedb.core.common.exception.ErrorMessage.Server.INCOMPATIBLE_JAVA_RUNTIME;
 import static com.vaticle.typedb.core.common.exception.ErrorMessage.Server.UNCAUGHT_EXCEPTION;
 import static com.vaticle.typedb.core.common.exception.ErrorMessage.Server.UNRECOGNISED_CLI_COMMAND;
-import static com.vaticle.typedb.core.server.logging.Logback.configure;
 import static com.vaticle.typedb.core.server.common.Util.getTypedbDir;
 import static com.vaticle.typedb.core.server.common.Util.printASCIILogo;
 
 public class TypeDBServer implements AutoCloseable {
     private static final Logger LOG = LoggerFactory.getLogger(TypeDBServer.class);
 
     protected final Factory factory;
-    protected final TypeDB.DatabaseManager databaseMgr;
+    protected final CoreDatabaseManager databaseMgr;
     protected final io.grpc.Server server;
-    protected final Config config;
+    protected final CoreConfig config;
     protected final boolean debug;
     protected TypeDBService typeDBService;
+    protected final CoreLogback logback;
 
-    private TypeDBServer(Config config, boolean debug) {
-        this(config, debug, new CoreFactory());
+    private TypeDBServer(CoreConfig config, boolean debug) {
+        this(config, debug, new CoreFactory(), new CoreLogback());
     }
 
-    protected TypeDBServer(Config config, boolean debug, Factory factory) {
+    protected TypeDBServer(CoreConfig config, boolean debug, Factory factory, CoreLogback logback) {
```

Comment:
Make `CoreLogback` injectable, allowing for a subclass to provide their own implementation.